conf/routes

```diff
@@ -91,6 +94,9 @@ POST   /v3/participants/self/identifiers @org.sagebionetworks.bridge.play.contro
 GET    /v3/participants/:userId                                        @org.sagebionetworks.bridge.play.controllers.ParticipantController.getParticipant(userId: String, consents: Boolean ?= true)
 POST   /v3/participants/:userId                                        @org.sagebionetworks.bridge.play.controllers.ParticipantController.updateParticipant(userId: String)
 DELETE /v3/participants/:userId                                        @org.sagebionetworks.bridge.play.controllers.ParticipantController.deleteTestParticipant(userId: String)
+GET    /v3/participants/:userId/sms/optout                             @org.sagebionetworks.bridge.play.controllers.SmsController.getOptOutSettings(userId: String)
+POST   /v3/participants/:userId/sms/optout                             @org.sagebionetworks.bridge.play.controllers.SmsController.setOptOutSettings(userId: String)
+GET    /v3/participants/:userId/sms/recent                             @org.sagebionetworks.bridge.play.controllers.SmsController.getMostRecentMessage(userId: String)
```

Comment:
I think opt-out status should be a separate entity because (a) we may or may not be required to track global opt-outs (b) if a user changes their phone number or deletes their phone number (such as a withdrawal), we may or may not be required to remember the opt-outs for that phone number (c) researchers aren't allowed to change a user's opt-out status.

The reason why these APIs in particular use userId and not phone number is because we log URLs, so the URL can't contain a phone number.

For the get API, it's currently scoped to admins, because it's intended to be used only by integration tests. I don't know if researchers currently want this (at least not in this format, and not in the Bridge Study Manager), so I didn't make the API available to researchers.

Reply:
Hmm. If these are tied to phone number, maybe just /v3/smsoptouts ? Put the phone number in the body message? We can live with this API, but if we need to remember these settings even if we delete the phone in an account, or delete an account entirely, it's not even related to a user account. Yes we would normally do POST /v3/smsoptouts/:phoneNumber but we can skip that for security purposes.

Reply:
I do also wonder how we'll answer the question "why isn't this user seeing SMS messages?" We would want people to be able to look in the UI and see the user's opt out status, even if it's not changeable. If they haven't opted out, then we would assume there's a problem and investigate further, I believe.

Reply:
Unfortunately, you can't use a request body with an HTTP GET. (Well, you can, but it's considered an anti-pattern, and some tools (like Postman) don't support it.)

As for how researchers can view opt-outs, there's already a separate sprint story for this.

Reply:
What will we do if the account is deleted? Are we just off the hook to persist opt out status at that point?

Reply:
These APIs are only used by integration tests. The actual opt-out handling comes in through the Twilio webhook.